conf/build-ghc-app.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 set -e
 export GOPATH="${HOME}/go"
-export GHC_APP_DIR="/github-contributions/ghc-app"
```

Comment:
where is it referenced? I think I was planning on adding a symlink from there to somewhere else, but then that ended up in the provision.yml